spec/models/comment_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+Rspec.describe Comment, type: :model do
```

Comment:
* I get this error when i run `rspec` command this error is caused by the typo `Rspec.describe` instead of `RSpec.describe` , kindly fix this 
![image](https://github.com/unorjikingsley/ROR-BlogApp/assets/52497246/76d4c688-5150-463a-b407-0f5d890f2a0e)